src/petals/server/server.py

```diff
@@ -174,14 +174,15 @@ def __init__(
                 first_block_index, last_block_index = map(int, map(str.strip, (first_block_index, last_block_index)))
             except Exception as e:
                 raise ValueError(f"Failed to parse `--block_indices {block_indices}`, must be start:end (e.g. 0:18)")
-            block_indices = range(first_block_index, last_block_index)
+            block_indices = list(range(first_block_index, last_block_index))
```

Comment:
Please leave `range()` here, so we have `range(0, 27)` in logs instead of a long line `[0, 1, ..., 26]`